scripts/build_all.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -ex
+
+for p in lockup multisig staking-pool staking-pool-factory voting whitelist
+do
+  pushd ../${p}
```

Comment:
```suggestion
 (cd ${p} && ./build.sh)
```
This will use a subprocess and avoid changing the parent script's pwd.